setup.py

```diff
@@ -239,7 +239,7 @@ def write_version(filename=os.path.join(*['airflow',
     + cassandra + mongo
 
 devel = [
-    'click',
+    'click==6.7',
```

Comment:
Sorry to be a pain, but can you make this change a separate PR.

Reply:
Ah NM someone has just created this as a PR. Rebase on to master in a few minutes.

Reply:
Good point. I see that seconds after you posted this kaxil did so: https://github.com/apache/incubator-airflow/pull/3962

Let's wait until this PR is merged, and I'll rebase it on master. :)